src/common/mclock_common.h

```diff
@@ -142,20 +143,18 @@ class MclockConfig {
   double osd_bandwidth_cost_per_io;
   double osd_bandwidth_capacity_per_shard;
   ClientRegistry& client_registry;
-  #ifndef WITH_CRIMSON
   MonClient *monc;
-  #endif
 public:
   #ifdef WITH_CRIMSON
   MclockConfig(CephContext *cct, ClientRegistry& creg,
                uint32_t num_shards, bool is_rotational, int shard_id,
-	       int whoami):cct(cct),
+	       int whoami, MonClient *monc):cct(cct),
```

Comment:
Do we still need two MclockConfig ctors? One for Crimson and one for classic? Seems like both of them are matching now and both could use the same ctor.

Reply:
done